docs/quickstart.rst

```diff
@@ -70,6 +70,19 @@ In one last terminal, start the admin:
 This will launch a command prompt where you can input admin commands to control and monitor many aspects of
 the FL process. Log in by entering ``admin`` for both the username and password.
 
+Custom Code in Example Apps
+===========================
+
+There are several ways to make :ref:`custom code <custom_code>` available to clients when using NVIDIA FLARE. Most
+hello-* examples use a custom folder within the FL application. Note that using a custom folder in the app needs to be
+:ref:`allowed <troubleshooting_byoc>` when using secure provisioning. By default, this option is disabled in the secure
+mode. POC mode, however, will work with custom code by default.
+
+In contrast, the CIFAR-10 example assumes that the learner code is already installed on the client's system and
```

Comment:
Add a link to the CIFAR-10 example. Might need some rewording as other examples (prostate, BraTS) also use this approach.